docs/source/guides/using_woodwork_with_dask_and_koalas.ipynb

```diff
@@ -294,7 +294,10 @@
     "When working with the LatLong logical type, Woodwork converts all LatLong columns to a standard format of a tuple of floats for Dask DataFrames and a list of floats for Koalas DataFrames. In order to do this, the data is read into memory, which may be problematic for large datatsets.\n",
     "\n",
     "### Integer Column Names\n",
-    "Woodwork allows column names of any format that is supported by the DataFrame. However, Dask DataTables do not currently support integer column names.\n",
+    "Woodwork allows column names of any format that is supported by the DataFrame. However, Dask DataFrames do not currently support integer column names.\n",
+    "\n",
+    "### Setting DataFrame Index\n",
+    "When specifying a Woodwork index with a pandas DataFrame, the index of the DataFrame will be set to the column specified as the index and the index column will also remain as a normal column in the DataFrame. When using Woodwork with Dask or Koalas DataFrames, the index of the DataFrame does not get set as it does with pandas, but rather the index column is kept as a normal column in the DataFrame.\n",
```

Comment:
Not totally sure that we need to describe the behavior in a Pandas DataFrame more than is necessary. (Though it might be useful to have in a guide elsewhere, and I'm not sure it's really documented anywhere)

Here's what I'm thinking might be a good way to simplify this:

> When specifying a Woodwork index with a pandas DataFrame, the underlying index of the DataFrame will be updated to match the column specified as the Woodwork index. When specifying a Woodwork index on a Dask or Koalas DataFrame, however, the underlying index will remain unchanged.

Reply:
I like it. Updated!